Packs/NetskopeV2/Integrations/NetskopeV2ForContributionToMarketplace/NetskopeV2ForContributionToMarketplace.py

```diff
@@ -0,0 +1,272 @@
+import traceback
+from typing import Any, Dict
+
+import demistomock as demisto  # noqa: F401
+import urllib3
+from CommonServerPython import *  # noqa: F401
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API"""
+
+    def get_lists(self):
+        return self._http_request(
+            method='GET'
```

Comment:
No url_suffix?

Reply:
no url_suffix is needed since the base_url joins the url configured in the instance with '/api/v2/policy/urllist/'